crates/bevy_ecs/src/query/state.rs

```diff
@@ -875,6 +875,58 @@ impl<D: QueryData, F: QueryFilter> QueryState<D, F> {
         }
     }
 
+    /// Return a [`PointQuery`] to get the query item for the given [`Entity`].
+    pub fn getter<'w, 's>(
```

Comment:
If this doesn't inline, there may result in the compiler unable to make assumptions abut the initialized point query.

This probably should also be renamed to `as_point_query`.